packages/compass-global-writes/package.json

```diff
@@ -62,7 +62,8 @@
     "react": "^17.0.2",
     "react-redux": "^8.1.3",
     "redux": "^4.2.1",
-    "redux-thunk": "^2.4.2"
+    "redux-thunk": "^2.4.2",
+    "@tanstack/table-core": "^8.14.0"
```

Comment:
I know that we're using it only in global-writes, but just from the perspective of being able to easier maintain matching versions of UI dependencies, do you mind moving this to compass-components and re-exporting from there? Otherwise it will be hard to track where to make version updates next time we update leafygreen table along other leafygreen deps

Reply:
sure, will do! I also asked LG team if they could re-export the methods or have the table as a peer dep, so that we don't end up with diverging versions.

Reply:
This would be ideal, yeah, thanks!

Reply:
moved the imports! this time I looked into LG and found out they actually do re-export tanstack table, I just didn't see it because some parts like the row type are overridden. now that I only need the type I thought it could even be just dev dep, but that didn't pass depcheck for some reason 🤔